great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -300,11 +300,20 @@ def __init__(
         # initialize runtime_environment as empty dict if None
         runtime_environment = runtime_environment or {}
         if self._ge_cloud_mode:
+            ge_cloud_base_url: Optional[str] = None
+            ge_cloud_access_token: Optional[str] = None
+            ge_cloud_organization_id: Optional[str] = None
+            if ge_cloud_config:
+                ge_cloud_base_url = ge_cloud_config.base_url
+                ge_cloud_access_token = ge_cloud_config.access_token
+                ge_cloud_organization_id = ge_cloud_config.organization_id
             self._data_context = CloudDataContext(
                 project_config=project_config,
                 runtime_environment=runtime_environment,
                 context_root_dir=context_root_dir,  # type: ignore[arg-type]
-                ge_cloud_config=ge_cloud_config,  # type: ignore[assignment,arg-type]
+                ge_cloud_base_url=ge_cloud_base_url,
```

Comment:
Any reason to pass these separately instead of as a GeCloudConfig object?

Reply:
just consistency with the current `DataContext` constructor - happy to alter this if we see fit but was aiming for parity over anything else.

Thankfully, this is not exactly public API (in the traditional sense) so we have a lot of flexibility.

Let me know your thoughts!

Reply:
That makes sense! Thanks for clarifying.